action.yml

```diff
@@ -34,6 +34,21 @@ runs:
         github_token: ${{ inputs.github-token }}
         labels: ${{ steps.extract-label.outputs.labels_to_add }}
 
+    - name: Remove legacy PR size labels
```

Comment:
maybe add a comment to remove when https://github.com/CodelyTV/pr-size-labeler/issues/96 is resolved